v22.1/row-level-ttl.md

```diff
@@ -0,0 +1,342 @@
+| `ttl_delete_batch_size`                                  | How many rows to [delete](delete.html) at a time. Default: 100. Minimum: 1.                                                                                                  | `sql.ttl.default_delete_batch_size` |
+| `ttl_delete_rate_limit`                                  | Maximum number of rows to be deleted per second (rate limit). Default: 0 (no limit).                                                                                         | `sql.ttl.default_delete_rate_limit` |
+| `ttl_range_concurrency`                                  | The Row-Level TTL queries split up scans by ranges, and this determines how many concurrent ranges are processed at a time. Default: 1. Minimum: 1.                          | `sql.ttl.default_range_concurrency` |
+| `ttl_row_stats_poll_interval`                            | If set, counts rows and expired rows on the table to report as Prometheus metrics while the TTL job is running. Unset by default, meaning no stats are fetched and reported. | N/A                                 |
+| `ttl_pause` <a name="param-ttl-pause"></a>               | If set, stops the TTL job from executing.                                                                                                                                    | N/A                                 |
+| `ttl_job_cron` <a name="param-ttl-job-cron"></a>         | Frequency at which the TTL job runs, specified using [CRON syntax](https://cron.help).                                                                                       | N/A                                 |
+| `ttl_automatic_column`                                   | If set, use the value of the `crdb_internal_expiration` hidden column.                                                                                                       | N/A                                 |
+
+For more information about TTL-related cluster settings, see [View TTL-related cluster settings](#view-ttl-related-cluster-settings).
+
+## Examples
+
+### Create a table with row-level TTL
+
+To specify a TTL when creating a table, use the SQL syntax shown below. For example, to create a new table with rows that expire after 15 minutes, execute a statement like the following:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE TABLE events (
+  id UUID PRIMARY KEY default gen_random_uuid(),
+  description TEXT,
+  inserted_at TIMESTAMP default current_timestamp()
+) WITH (ttl_expire_after = '3 minutes');
+~~~
+
+~~~
+CREATE TABLE
+~~~
+
+Insert some data; it should work as expected:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+INSERT INTO events (description) VALUES ('a thing'), ('another thing'), ('yet another thing');
+~~~
+
+~~~
+INSERT 3
+~~~
```

Comment:
maybe add a `SELECT *, crdb_internal_expiration FROM events` to show the rows and their expirations

Reply:
Did it!